src/tests/compiler/expression/object/propertyAssignmentTests.ts

```diff
@@ -1,7 +1,8 @@
-import { expect } from "chai";
-import { PropertyAssignment, ShorthandPropertyAssignment } from "../../../../compiler";
+import { expect, assert } from "chai";
+import { PropertyAssignment, ShorthandPropertyAssignment, ObjectLiteralExpression, ObjectLiteralElementLike, Node } from "../../../../compiler";
 import { SyntaxKind } from "../../../../typescript";
-import { getInfoFromText } from "../../testHelpers";
+import { getInfoFromText, getInfoFromTextWithDescendant } from "../../testHelpers";
+import { TypeGuards } from "../../../../utils/TypeGuards";
```

Comment:
You could change this to `"../../../../utils"`. In the future I'd like to get `rootDirs` working so that so many relative paths aren't necessary.